Build bug: 'ICorProfilerCallback8': base class undefined

Build instructions should have this workaround added: https://github.com/Microsoft/BPerf/issues/13

I get the errors like `undefined reference to 'IID_ICorProfilerInfo8'` while building **ReJITEnterLeaveHooks** example with ubuntu 20.04, could this be the same error?

Does someone have an idea how to solve it?

--- a/ProfilingAPI/ReJITEnterLeaveHooks/ILRewriter.cpp
+++ b/ProfilingAPI/ReJITEnterLeaveHooks/ILRewriter.cpp
@@ -491,7 +491,6 @@ public:
                 m_pOutputBuffer[offset++] = (opcode & 0xFF);
             }
 
-            assert(pInstr->m_opcode < _countof(s_OpCodeFlags));
             BYTE flags = s_OpCodeFlags[pInstr->m_opcode];
             switch (flags)
             {

--- a/ProfilingAPI/ReJITEnterLeaveHooks/build.sh
+++ b/ProfilingAPI/ReJITEnterLeaveHooks/build.sh
@@ -16,6 +16,6 @@ printf '  Building %s ... ' "$Output"
 CXX_FLAGS="$CXX_FLAGS --no-undefined -Wno-invalid-noreturn -fPIC -fms-extensions -DBIT64 -DPAL_STDCPP_COMPAT -DPLATFORM_UNIX -std=c++11"
 INCLUDES="-I $CORECLR_PATH/src/pal/inc/rt -I $CORECLR_PATH/src/pal/prebuilt/inc -I $CORECLR_PATH/src/pal/inc -I $CORECLR_PATH/src/inc -I $CORECLR_PATH/bin/Product/$BuildOS.$BuildArch.$BuildType/inc"
 
-clang++ -shared -o $Output $CXX_FLAGS $INCLUDES ClassFactory.cpp CorProfiler.cpp dllmain.cpp ILRewriter.cpp
+clang++ -shared -o $Output $CXX_FLAGS $INCLUDES ClassFactory.cpp CorProfiler.cpp dllmain.cpp ILRewriter.cpp $CORECLR_PATH/src/pal/prebuilt/idl/corprof_i.cpp
 
 printf 'Done.\n'

